contribute/release-policy.md

```diff
@@ -0,0 +1,84 @@
+---
+id: release-policy
+title: Release policy
+---
+
+## Release semantics
```

Comment:
Should it be semantic version?

Reply:
I lower the relation to semver. Please see the latest commit f29a51cc9178f8a02acf5eb9d99d56e1c19abb1a